models/vista3d/configs/inference_trt.json

```diff
@@ -0,0 +1,12 @@
+{
+    "imports": [
+        "$import glob",
+        "$import json",
+        "$import pathlib",
+        "$import os",
+        "$import scripts",
+        "$import numpy as np",
+        "$from monai.networks import trt_wrap"
+    ],
+    "network": "$trt_wrap(@network_def.to(@device), @bundle_root + '/models/model.pt', submodule=['image_encoder.encoder', 'class_head'])"
```

Comment:
thanks for the changes.
Currently model zoo CI excludes vista3d bundle's torchscript test.
After trt_wrap merging into MONAI, I can modify: https://github.com/Project-MONAI/model-zoo/blob/dev/ci/verify_bundle.py#L339 to enable it.